CONTRIBUTING.md

```diff
@@ -235,8 +235,11 @@ locations.
     └─ cli/
        ├─ mime_type_map.h           # Generated by `sapi/cli/generate_mime_type_map.php`
        └─ ...
+    └─ embed/
+       ├─ php-embed.pc.in           # pkg-config metadata file
+       └─ ...
     └─ ...
- ├─ scripts/                        # php-config, phpize and internal development scripts
+ ├─ scripts/                        # php-config, phpize, pkg-config, and internal development scripts
```

Comment:
This might better be `pkg-config metadata file` or maybe just `.pc`, or something like this.